root/sbin/e-smith/rsync-upgrade

```diff
@@ -118,6 +118,9 @@ fi
 # Avoid deleting backup history directory
 echo "/var/lib/nethserver/backup/history" >> $exclude_f
 
+# Avoid deleting restore history directory
```

Comment:
What is written into `restore/`? Is "history" appropriate?

```suggestion
# Avoid deleting restore directory
```

Reply:
The dir contains a file for each backup, each containing a list of files that can be restored.
I'd call that a history. :-)